components/automate-cli/cmd/chef-automate/certRotate.go

```diff
@@ -574,6 +684,53 @@ func (c *certRotateFlow) copyAndExecute(ips []string, sshUtil SSHUtil, timestamp
 	return nil
 }
 
+// copyAndExecuteConcurrently will copy the toml file to each required nodes concurrently and then execute the set of commands.
+func (c *certRotateFlow) copyAndExecuteConcurrentlyToFrontEndNodes(ips []string, sshConfig sshutils.SSHConfig, timestamp string, remoteService string, fileName string, scriptCommands string, flagsObj *certRotateFlags, printCertRotateOutput func(sshutils.Result, string, *cli.Writer)) error {
```

Comment:
Why are passing this as a parameter?
Can you remove it?

Reply:
done